docs/projects/ci-integration.md

```diff
@@ -81,3 +82,78 @@ run:
 ```
 
 See the full example in [**https://gitlab.com/yhaviv/test2**](https://gitlab.com/yhaviv/test2)
+
+## Using Jenkins Pipeline
+
+When using [Jenkins Pipeline](https://www.jenkins.io/doc/book/pipeline/) you need to set up the credentials/secrets in Jenkins and 
+and update the script `Jenkinsfile` in your codebase. You can trigger the Jenkins pipeline either through Jenkins triggers or through the GitHub webhooks. 
+
+Example `Jenkinesfile` that is invoked when you start a Jenkins pipeline (via triggers or GitHub webhooks):
+
+```Groovy
+pipeline {
+   agent any
+    environment {
+      RELEASE='1.0.0'
+      PROJECT_NAME='project-demo'
+    }
+   stages {
+      stage('Audit tools') {
+         steps{
+            auditTools()
+         }
+      }
+      stage('Build') {
+            environment {
+               MLRUN_DBPATH='https://mlrun-api.default-tenant.app.us-sales-341.iguazio-cd1.com'
+               V3IO_ACCESS_KEY=credentials('V3IO_ACCESS_KEY')
+               V3IO_USERNAME='xingsheng'
+               V3IO_API="https://default-tenant.app.us-sales-341.iguazio-cd1.com:8444"
+            }
+            agent {
+                docker {
+                    image 'mlrun/mlrun:1.0.0'
+                }
+            }
+            steps {
+               echo "Building release ${RELEASE} for project ${PROJECT_NAME}..."
+               sh 'chmod +x build.sh'
+               withCredentials([string(credentialsId: 'an-api-key', variable: 'API_KEY')]) {
+                  sh '''
+                     ./build.sh
+                  '''
+               }
+            }
+        }
+        stage('Test') {
+            steps {
+               echo "Testing release ${RELEASE}"
+            }
+        }
+   }
+   post {
+      success {
+         slackSend channel: '#builds',
+                   color: 'good',
+                   message: "Project ${env.PROJECT_NAME}, success: ${currentBuild.fullDisplayName}."
+      }
+      failure {
+         slackSend channel: '#builds',
+                   color: 'danger',
+                   message: "Project ${env.PROJECT_NAME}, FAILED: ${currentBuild.fullDisplayName}."
+      }
+   }
+}
+
+void auditTools() {
+   sh '''
+      git version
+      docker version
+   '''
+}
+```
+After the Jenkins pipeline is complete, you will be able to see the MLRun job in the Iguazio Platform GUI like the screenshot below:
```

Comment:
After the Jenkins pipeline is complete, you can see the MLRun job in the Iguazio Platform GUI similar to the screenshot below:

Reply:
changed the wording

Reply:
Can you just add period at the end of the two last sentences:

- ...MLRun UI**.**
- See the full example in <link>**.**

Thanks